app/views/components/_no_sam_verification_header.html.erb

```diff
@@ -10,3 +11,8 @@
     <p>*NOTE: We periodically check if all users are listed in SAM.gov. It may be possible that you registered after the most recent check. However, we will always run a check at least once per day, so you may have to wait up to 24 hours between registration and bidding.</p>
   </div>
 </div>
+
+<div class='usa-alert usa-alert-warning warning-show-trigger' style='display:none'>
```

Comment:
I've been looking at either how they did it on the [CFPB beta page](http://beta.consumerfinance.gov/) or as described in [this list of practical examples](http://heydonworks.com/practical_aria_examples/#progressive-collapsibles). There is also [the spec](https://www.w3.org/TR/wai-aria/), but...